x-pack/test/saved_object_tagging/functional/tests/visualize_integration.ts

```diff
@@ -95,7 +98,10 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await selectSavedObjectTags('tag-1');
 
         await testSubjects.click('confirmSaveSavedObjectButton');
+        await testSubjects.existOrFail('saveVisualizationSuccess');
+
         await PageObjects.visualize.gotoVisualizationLandingPage();
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
Suggestion: There could be a lot of reasons for waiting for a DOM element to be rendered. What if we increase `timeout` in `selectFilterTags` method instead?

Reply:
TBH after 100 runs against my local I couldn't reproduce the flaky failure even once. The failure message indicates that for some reason, 1. the tag filter was not applied (`Visualization 2 (tag-2)` is present in the table) and 2. the new vis was not created (`My new markdown viz` is not), and I don't really how we could come as far as as the last assertion in that state without failing before in the test. So I kinda do the best I could with additional synchronizations / waiting steps to see if that would resolves it on CI (renamed the PR accordingly).

I can do what you suggest though

Reply:
@pgayvallet Have you tried to leverage https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/build
to run a flaky test on CI?

Reply:
I did not. Can this be ran against a PR branch?

Reply:
you can configure it
![2020-11-10_12-30-14](https://user-images.githubusercontent.com/3198181/98655460-c92ccb80-233f-11eb-81e7-fcbadfd0afe7.png)

Reply:
First run had a failure on another test - https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/978/testReport/junit/X-Pack%20Saved%20Object%20Tagging%20Functional%20Tests/x-pack_test_saved_object_tagging_functional_tests_dashboard_integration%C2%B7ts/Kibana_Pipeline___agent_2___saved_objects_tagging___functional_tests_dashboard_integration_creating_allows_to_create_a_tag_from_the_tag_selector/

but the error message was a little more explicit on that one. trying something else by waiting for the table to load - https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/983/